vendor/github.com/giantswarm/k8scloudconfig/v_0_1_0/master_template.go

```diff
@@ -854,11 +855,12 @@ write_files:
           serviceAccountName: nginx-ingress-controller
           containers:
           - name: nginx-ingress-controller
-            image: {{.Cluster.Kubernetes.IngressController.Docker.Image}}
+            image: quay.io/kubernetes-ingress-controller/nginx-ingress-controller:0.9.0-beta.17
```

Comment:
Can you confirm we want these ingress changes to be vendored?

Reply:
Very good point, checked with @puja108 and it's ok to have this in place for new clusters.